lib/Cake/Utility/String.php

```diff
@@ -355,6 +355,23 @@ public static function wrap($text, $options = array()) {
  * @return string Formatted text.
  */
 	public static function wordWrap($text, $width = 72, $break = "\n", $cut = false) {
+		$paragraphs = explode($break, $text);
+		foreach ($paragraphs as &$paragraph) {
+			$paragraph = String::_wordWrap($paragraph, $width, $break, $cut);
```

Comment:
But the user has requested the text to be wrapped to a specific length. I guess it is debatable whether the result should be a block of fully justified text though.